src/animation.js

```diff
@@ -31,7 +32,7 @@ class AnimationHandle {
 		}
 		if (this.paused === false) {
 			let dt = time - this._lastChange;
-			let x = this._lastChangeX + this.speed * dt;
+			let x = this._lastChangeX + (this.speed * this._animationNaturalSpeed) * dt;
```

Comment:
`naturalSpeed` is an intrinsic property of `Animation`. We can move the multiplication to `invokeAnimation`.

Assuming `naturalSpeed` is immutable:
```
x = x0 + (speed + naturalSpeed) * dt
   = (speed_0 + naturalSpeed) * dt_0 + (speed_1 + naturalSpeed) * dt_1 + ... + (speed_n + naturalSpeed) * dt_n
   = naturalSpeed * (speed_0 * dt_0 + speed_1 * dt_n + ... + speed_2 * dt_n)
```
So it's no difference whether we put the multiplication in `AnimationHandle` or `invokeAnimation`.

But when we invoke the animation directly (not using `AnimationHandle`), `naturalSpeed` will be considered if we put the multiplication in `invokeAnimation`.

Reply:
No, it's `speed * naturalSpeed`, multiplication but not plus.

Reply:
Sorry, I just wrote it wrong. Now I have corrected it.